packages/patternfly-3/patternfly-react-extensions/src/components/CatalogTile/CatalogTile.js

```diff
@@ -4,38 +4,66 @@ import classNames from 'classnames';
 
 import CatalogTileBadge from './CatalogTileBadge';
 
-const CatalogTile = ({
-  id,
-  className,
-  featured,
-  href,
-  onClick,
-  iconImg,
-  iconClass,
-  badges,
-  title,
-  vendor,
-  description,
-  maxDescriptionLength,
-  truncateDescriptionFn,
-  ...props
-}) => {
-  const classes = classNames('catalog-tile-pf', { featured }, className);
-
-  const defaultTruncateDescription = (text, max) => {
-    if (max === -1 || typeof text !== 'string' || text.length <= max) {
-      return text;
+const defaultTruncateDescription = (text, max) => {
+  if (max === -1 || typeof text !== 'string' || text.length <= max) {
+    return text;
+  }
+
+  return (
+    <React.Fragment>
+      {text.substring(0, max - 3)}
+      &hellip;
+    </React.Fragment>
+  );
+};
```

Comment:
Notes: since we talked about this one, alternative approaches included going with a static version of this method, this way works too